app/views/organizations/staff/_form.html.erb

```diff
@@ -0,0 +1,61 @@
+<!-- form -->
+<%= form_with model: staff, url: '/staff', method: :post do |form| %>
+  <%= form.hidden_field :organization_id, value: current_user.staff_account.organization_id %>
```

Comment:
```suggestion
```
You shouldn't need this because `acts_as_tenant` automatically puts the current tenant on `.new`.
```
ActsAsTenant.with_tenant(Organization.find(id: 5)) do 
     StaffAccount.new # will automatically put organization_id=5, assuming acts_as_tenant is included in the model
end
```